Button

alucard822 opened the pull request:
Added button and background

alucard822 committed:
Добавил кнопку и фон

AnastasiaKallisto commented on website/index.html:
```diff
@@ -0,0 +1,91 @@
+  <body>
+    <nav class="menu">
+      <div class="container">
+        <div class="menu-burger__header">
```
P.S. Поправлю этот момент сама, просто сообщаю

![image](https://github.com/user-attachments/assets/76167366-c89b-4486-af9a-808d10f2f485)

AnastasiaKallisto replied:
Другой тип меню - выпадающее. На мой взгляд, пунктов меню слишком мало, чтобы делать большую вертикальную полоску.

Задержка скрытия 0.3 секунды

![image](https://github.com/user-attachments/assets/56e5cca5-e188-4341-a4b0-1f45d79e10a1)

AnastasiaKallisto committed:
визуальные правки

alucard822 committed:
Добавил увеличении кнопки

AnastasiaKallisto commented on website/css/style.css:
```diff
@@ -7,6 +7,8 @@
 body {
   background-color: #ccd7d8;
   font-size: 20px;
```
Это свойства body - основной страницы, а не кнопки

AnastasiaKallisto commented on website/css/style.css:
```diff
@@ -44,6 +46,11 @@ body {
   cursor: pointer;
```
А вот реальные свойства кнопки зашиты где-то здесь. Только они связаны с шириной шторки, которая нам будет не нужна. Рекомендую писать код с нуля для этой задачи, а не пытаться переписать именно это решение. 

15px достаточно, как выяснилось

alucard822 committed:
Изменил кнопку

AnastasiaKallisto commented on website/index.html:
```diff
@@ -8,10 +8,16 @@
           <span></span>
-        </div>
+          <div class="dropdown-content">
+            <a href="#"><img src="https://img.freepik.com/free-photo/studio-background-concept-abstract-empty-light-gradient-purple-studio-room-background-product_1258-54682.jpg?t=st=1737564526~exp=1737568126~hmac=6bcf0bdfb0d0e2f7ee006216238e194b8a0e34b9229c297cdc34c566ec79643d&w=826" alt="Фиолетовый">ИГРАТЬ</a>
```
Картинки лучше скачать, взять локальную png, можешь в эту же папку положить пока, пусть валяется рядом

AnastasiaKallisto commented on website/css/style.css:
```diff
@@ -69,39 +59,83 @@ body {
-.menu.active .menu-burger__header::before {
+.menu.active .menu-dropbtn::before {
   top: 50%;
   transform: translate(-50%, -50%) rotate(45deg);
```
Лодочный якорь? (неиспользуемый/бесполезный код?)
Не могу представить зачем здесь сейчас вращение на 45 градусов

AnastasiaKallisto commented on website/css/style.css:
```diff
@@ -69,39 +59,83 @@ body {
 
-.menu .menu-burger__header::after {
+.menu .menu-dropbtn::after {
   top: calc(50% + 8px);
```
Не сильно вникала где именно, напишу здесь
Должен быть отступ вниз от черной кнопки, и примерно 0.3 секунды перед ее появлением
Меню появляется при клике, не при наведении
Картинки выровнять по левой стороне, как и надписи
До левого края отступ примерно 10 пикселей
И от картинки до надписи пункта меню тоже 10 пикселей
Картинка должна быть выровнена по центру по вертикали в рамках своего пункта меню 
(сейчас выравнивание картинки по низу текста)

![image](https://github.com/user-attachments/assets/32216645-96aa-4a36-8a16-e29b47443829)

alucard822 committed:
Распределил по папкам css

alucard822 committed:
Остались небольшие моментики

alucard822 committed:
Поправил наименования

AnastasiaKallisto committed:
- GW-14: убрала некоторые css

AnastasiaKallisto committed:
+ GW-14: четыре html страницы + переименование заголовка

AnastasiaKallisto committed:
* GW-14: переименования стилей

AnastasiaKallisto committed:
* GW-14: убрала div из внутрянки кнопки

начала выносить то, что относится напрямую к внешнему виду кнопки,
в отдельный файл

AnastasiaKallisto committed:
* GM-14: убрала футер (если понадобится, вернем)

убрала предположительно бесполезный тег nav и стили, которые с ним связаны
поправила свойство header (из-за него кнопка съезжала вниз)

AnastasiaKallisto committed:
* GW-14: скругленное меню

скругленные ссылки
выравнивание ссылок по высоте

AnastasiaKallisto committed:
* GW-14: Правки, чтобы меню

правильно закрывалось и открывалось

AnastasiaKallisto committed:
= GW-14: переименования и опечатки

AnastasiaKallisto committed:
*GW-14: отступы в выпадающем меню

alucard822 committed:
Активный пункт меню

alucard822 committed:
Импортирование меню во все страницы

AnastasiaKallisto committed:
Merge branch 'master' into frontend/alucard822/guitarwebsite-14-menu-html
